sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -1572,7 +1589,9 @@ class Dataset[T] private[sql](
    * @since 2.0.0
    */
   @scala.annotation.varargs
-  def select(col: String, cols: String*): DataFrame = select((col +: cols).map(Column(_)) : _*)
+  def select(col: String, cols: String*): DataFrame = withOrigin {
```

Comment:
I don't think this is helpful -- the underlying `select` already has a `withOrigin` call, no?

Reply:
We are reverting it in https://github.com/apache/spark/pull/44501